pkg/okctl/okctl.go

```diff
@@ -30,8 +30,6 @@ import (
 
 	"github.com/oslokommune/okctl/pkg/config/state"
 
-	"github.com/oslokommune/okctl/pkg/api/core/store/noop"
```

Comment:
What were we using noop for before, and what replaced them anyway?

Reply:
It was introduced when we were transitioning from storing things on the filesystem serverside, just a way of not having to refactor _everything_ at once